lib/alarm.dart

```diff
@@ -175,12 +194,14 @@ class Alarm {
   }
 
   /// Returns all the alarms.
-  static List<AlarmSettings> getAlarms() => AlarmStorage.getSavedAlarms();
+  static Future<List<AlarmSettings>> getAlarms() =>
+      AlarmStorage.getSavedAlarms();
 
   /// Reloads the shared preferences instance in the case modifications
   /// were made in the native code, after a notification action.
   static Future<void> reload(int id) async {
-    await AlarmStorage.prefs.reload();
+    // TODO(orkun1675): Remove this function and publish stream updates for
```

Comment:
TODO

Reply:
This is out of scope of this migration PR. I added a TODO here because updating the `ringStream` when an alarm stops ringing would be great feature addition and should be easy to do in a follow-up PR.